src/sortingtraining/utils/SortingUtil.java

```diff
@@ -18,10 +18,8 @@ public SortingUtil(SortingStrategyResolver sortingStrategyResolver) {
     public int [] sort(int[] numbers, Strategy strategy) {
 
         SortingStrategy sortingStrategy = getResolve(strategy);
-
         int[] sorted = sortingStrategy.sort(numbers);
-
-        System.out.println(Arrays.toString(sorted));
+        System.out.println(Arrays.toString(sorted) + "\n");
```

Comment:
I don't know if you know but `ln` is for "line". So it's printing line that ends with new line feed. Of course if you don't want end line feed you may use `System.out.print`, and leave the end line here because you'd like two end line feeds. No issue here, its just for your info.

Reply:
100% aware of that!